events/events-team/comms/README.md

```diff
@@ -1,55 +1,191 @@
 - [Documentation](#documentation)
 
-## Overview
+## KCS Comms Lead Overview
+
+A KCS Comms Lead should plan to be involved with KCS planning for at least a year (2 KCS events), first as lead (for 1 or more events), and then as an emeritus mentor to the succeeding Comms lead (for 1 or more events). The workload of being an Emeritus Comms Lead is light, but they are still actively involved.
+
+A KCS Comms Lead holds that role from the beginning of planning for a specific KCS event until all activities for that event are completed, including post-event activities such as event retros and updating the handbook (this doc). This timeframe is the "cycle" for that KCS event.
 
 As Communications Lead, you are responsible for the overall communications
 before, during, and after the event. The role also includes ensuring all non-technical content (web, mail,
 physical prints) is correct and informative, and supporting the other roles
 (registration, content) as needed.
 
+A Comms Lead should plan to attend their KCS event in-person. Onsite, a Comms lead acts as the contact person for any comms needed onsite, such as messaging about room or registration changes that may occur onsite. In the event that a KCS Comms Lead is unable to attend in-person, the lead is responsible for designating an in-person comms point of contact.
+
+When their cycle is complete, a KCS Comms Lead becomes "emeritus." The most recent Emeritus Comms Lead should plan to be actively involved in the following KCS cycle as a mentor to the next KCS Comms Lead.
+
 Time Commitment:
 - 1-3 hours a week from 0-1.5 months in
 - 3-5 hours a week from 1.5 months-to event
+- 1-3 hours post-event for retros and handbook updates
+- 1-2 hours per month through the following KCS cycle.
 
-## Skills and Qualifications
+### Skills and Qualifications
 
+- A KCS Comms Lead must have [Kubernetes Org Membership]. (They should not plan to pursue membership during the cycle.)
+- A KCS Comms Lead should plan to attend their KCS event in-person
+- Successful completion of a past cycle as a KCS Comms Shadow, or equivalent experience
 - Good grasp on general comms activities surrounding events
-- Understanding of common forms of communications
-- Event planning experience
-- If working on the website, you'll need HTML/CSS/JavaScript experience
+- Understanding of common forms of contributor communications including Slack, email, and contributor-owned social media accounts
 - A healthy emoji vocabulary!
 
-## Responsibilities
+### Responsibilities
 
-- Create a communication schedule regarding the event - what, when, how, who
+- Create a Comms Team plan for the cycle
+  - Create a KCS Comms Team plan using the [KCS Comms Team Plan Template]
+    - Shadow requirements
+    - Volunteer requirements
+    - Associated activities such as comms and meeting updates
+    - Update the included rough comms timelines
+- Create a doc for comms drafts using the [KCS Comms Draft Doc Template]
```

Comment:
Do we want to mention about Award / logo designs?

Reply:
This should be owned by the Design lead. Which does not currently exist, but should in the future. I'll need to create a new role handbook.